test/Influencers/InfluencersTest.php

```diff
@@ -292,6 +292,146 @@ public function testAddTwitterMissingParams()
         Traackr\Influencers::addTwitter([]);
     }
 
+    /**
```

Comment:
These tests are mainly copied from existing Twitter based tests. I'll point out differences below. If we ever deprecate/remove the old twitter methods, we can purge the duplication.